linode/manager
# M3-5859: Upgrade Cypress to v10

## Description

**What does this PR do?**

This upgrades Cypress from v9.x to v10.x.

Some changes were made to Cypress's config and directory structure to accommodate its new component testing functionality.

## How to test

**What are the steps to reproduce the issue or verify the changes?**

Confirm that Cypress tests work as expected:
```
yarn cy:debug
```

```
yarn cy:run
```
